components/date-picker/date-picker.component.ts

```diff
@@ -369,6 +369,8 @@ export class NzDatePickerComponent implements OnInit, OnChanges, OnDestroy, Afte
       this.focus();
       this.updateInputWidthAndArrowLeft();
     });
+
+    this.elementRef.nativeElement.addEventListener('mousedown', this.onMouseDown);
```

Comment:
Could you add some comments about the issue to this?

Reply:
ok, i have added comments for this